### ISSUE TITLE
Misplaced inner icon when rendering using v-for

### ISSUE THREAD

**Issue by thejcannon** [Justineo/vue-awesome#145]
I have a template which will render possibly nested icons like so:
```html
<v-icon>
  <v-icon
    v-for="icon in info.icons"
    :key="`${info.title}_${icon.name}`"
    :name="icon.name"
    :scale="icon.scale || 1.25"
  />
</v-icon>
```

When I use an `info` that has multiple icons, it looks great. However, if I change `info` from having 1 icon to having multiple, the `x`/`y` of the second inner icon is different than its "looks great" counterpart. (The `x`/`y` in the "looks great" case is `false`/`false`, and in the changed case is `2.8`/`2.8`).

(I tried to make a codepen, but couldn't find anywhere that provides `vue-awesome.js` by itself. I'd be happy to provide a working example if you point me int he right direction to get one working)

E.g.
If info has "check" and "ban" I get this (with some scaling/coloring)
![image](https://user-images.githubusercontent.com/3956745/56672997-102f6f00-667d-11e9-801c-3155550e843f.png)
If info goes from just having "question" to having "check"/"ban" I get this:
![image](https://user-images.githubusercontent.com/3956745/56673044-2b01e380-667d-11e9-865c-bbeabffd7a5c.png)

**Comment by Justineo** [comment]
You can fork this pen -> https://codepen.io/Justineo/pen/YMMPQd

**Comment by thejcannon** [comment]
Thanks! Here's the working example with the bug: https://codepen.io/anon/pen/KYYNJB

**Comment by Justineo** [comment]
Fixed in 3.5.2.
